src/commands/synthetics/run-tests-command.ts

```diff
@@ -143,6 +144,7 @@ export class RunTestsCommand extends Command {
   private tunnel = Option.Boolean('-t,--tunnel', {
     description: `Use the ${$3('Continuous Testing Tunnel')} to execute your test batch.`,
   })
+  // TODO SYNTH-12989: Clean up deprecated `variableStrings` in favor of `variables` in `defaultTestOverrides`.
   private variableStrings = Option.Array('-v,--variable', {description: 'Pass a variable override.'})
```

Comment:
Can you do the same as [here](https://github.com/DataDog/datadog-ci/pull/1356/files#diff-083f09b898f286ae51bdc48ee28d591f3a567f3a3e295a19d14ab19e009f0d9eR134-R139)?

Namely:
- mark it as `@deprecated`
- change its description to a deprecation notice describing the new way